src/ui/ResponsiveComponents/ResponsiveFacetColumn.ts

```diff
@@ -71,12 +70,12 @@ export class ResponsiveFacetColumn implements IResponsiveComponent {
     const facetsMobileModeComponent = this.facetsMobileModeComponent;
     if (!facetsMobileModeComponent) {
```

Comment:
This technically can't happen anymore, should we log an error instead?

Reply:
Yeah, "just in case" :) AKA: If for some reason the dynamic import does not resolve quick enough ?